en/user_guide/user.rst

```diff
@@ -0,0 +1,53 @@
+User, Role and Group
+====================
+
+Open Orchestra offers the possibility to manage different users with different rights.
+
+User Management
+---------------
+
+The users page allows to search, create, edit and delete users.
+
+At the creation, some fields are required :
+
+* **first Name**
+* **last Name**
+* **e-mail**
+* **username**
+* **password**
+
+Once the user is created, you can add his default language and his groups.
+
+Role Management
+---------------
+
+A role gives one type of authorization for a user.
+
+Roles contains:
+
+* **name**
+* **descriptions**: describe role in every languages to display in admin panel
```

Comment:
``to display in admin panel`` It sounds to me that something is missing.

You could suppress this part.

Or, the description is used when you have a form containing roles and the role description is displayed